src/test/java/co/com/api/test/modelBody/DataModel.java

```diff
@@ -0,0 +1,32 @@
+package co.com.api.test.modelBody;
+
+public class DataModel {
+
+    private String name;
+    private String age;
+    private String city;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getAge() {
+        return age;
+    }
+
+    public void setAge(String age) {
+        this.age = age;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
```

Comment:
use lombok library in order to avoid boilerplate code

Reply:
Done